recipe/build.sh

```diff
@@ -29,4 +29,8 @@ fi
 
 export DAAL4PY_VERSION=$PKG_VERSION
 export MPIROOT=${PREFIX}
+
 ${PYTHON} setup.py install $ARGS
+
+# Confirm that the _onedal_py_host .so file was generated.
+test -f onedal/_onedal_py_host*.so || "ERROR: Failed to generate onedal/_onedal_py_host*.so"
```

Comment:
Would this be better as a test entry in `meta.yaml`?

Reply:
Thanks for this, Scott. I initially had the test in the build.sh so it would fall over quickly when the file was not present. But the test section makes more sense now.